code/modules/modular_computers/hardware/hard_drive.dm

```diff
@@ -179,7 +179,7 @@
 	desc = "An efficient SSD for portable devices developed by a rival organisation."
 	power_usage = 8
 	max_capacity = 70
-	var/datum/antagonist/traitor/traitor_data // Syndicate hard drive has the user's data baked directly into it on creation
```

Comment:
was this compatible with traitors before?

Reply:
previously it required you to be a traitor, but i changed it to OPFOR datum since this is OPFOR equipment and it's overall easier to handle